### ISSUE TITLE
Making a new project error

### ISSUE THREAD

**Issue by Jst28** [richdynamix/magestead#72]
PHP Warning:  file_put_contents(/home/johnst28/.composer/auth.json): failed to open stream: No such file or directory in /home/johnst28/.config/composer/vendor/richdynamix/magestead/src/Magestead/Helper/Options.php on line 200

**Comment by richdynamix** [comment]
Which host OS do you have? It would appear you are using a Linux flavour that uses the `XDG_CONFIG_HOME`. Currently Magestead does not support this flavour of Linux or Windows machines. This is something I am hoping to start working  soon.

**Comment by Jst28** [comment]
Linux Mint 17

**Comment by markorapaic** [comment]
Just for the record, i'm getting this same error and i'm on a mac running El Capitan. I've installed composer via homebrew and it runs globally without any issues. I've also checked that location, and the `auth.json` file is definitely there.

`PHP Warning:  file_put_contents([home] /Users/markorapaic/.composer/auth.json): failed to open stream: No such file or directory in /Users/markorapaic/.composer/vendor/richdynamix/magestead/src/Magestead/Helper/Options.php on line 201`
